global_planner/src/orientation_filter.cpp

```diff
@@ -97,6 +97,28 @@ void OrientationFilter::processPath(const geometry_msgs::PoseStamped& start,
                 }
             }
             break;
+        case LEFTRIGHTWARD:
+            for (int i = 0; i < n - 1; i++) {
+                setAngleBasedOnPositionDerivative(path, i);
+            }
+            if (n > 2) {
+                const int num_skips = n >= 5 ? 2 : 1;
+                const double start_path_theta = tf2::getYaw(path[num_skips].pose.orientation);
+                const double goal_path_theta = tf2::getYaw(path[n - 1 - num_skips].pose.orientation);
+                const double start_orientation = tf2::getYaw(start.pose.orientation);
+                const double rotation_to_path = angles::shortest_angular_distance(start_orientation, start_path_theta);
+                const double rotation_to_goal = angles::shortest_angular_distance(start_orientation, goal_path_theta);
```

Comment:
Q/F: shouldn't this be sth like
```suggestion
                const double rotation_to_goal = angles::shortest_angular_distance(goal_path_theta, tf2::getYaw(path[i-1].pose.orientation));
```
?

Reply:
actually, I should just copy from the bidirectional one. Fixed